src/coreclr/jit/gentree.cpp

```diff
@@ -12356,7 +12370,9 @@ void Compiler::gtDispLeaf(GenTree* tree, IndentStack* indentStack)
         case GT_MEMORYBARRIER:
         case GT_PINVOKE_PROLOG:
         case GT_JMPTABLE:
+#ifdef SWIFT_SUPPORT
         case GT_SWIFT_ERROR:
+#endif // SWIFT_SUPPORT
```

Comment:
Seeing this I would prefer we don't put the definition of this node under ifdef. Doesn't feel worth it to add to the ifdef soup when it's defined for our main target platforms anyway and these likely aren't associated with any TP cost.